src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/BlobProcessorTask.java

```diff
@@ -390,7 +390,8 @@ private void sendErrorMessageToQueue(
                 null,
                 null,
                 errorCode,
-                cause.getMessage()
+                cause.getMessage(),
+                "bulk_scan_processor"
```

Comment:
Should be a constant

Reply:
I tried it making a constant but it's not used anywhere else and saw a warning to make it local variable.